lib/features/composer/presentation/composer_view_web.dart

```diff
@@ -813,6 +818,42 @@ class ComposerView extends GetWidget<ComposerController>
                   isSelected: controller.richTextWebController.isTextStyleTypeSelected(textType),
                   tooltip: textType.getTooltipButton(context),
                   onTap: () => controller.richTextWebController.toggleCodeView());
+            case RichTextStyleType.headerStyle:
```

Comment:
as the new design
<img width="628" alt="image" src="https://user-images.githubusercontent.com/6462404/180456307-38d14a51-a8b3-4927-919c-0ee625b89d90.png">. 

Please move it to the toolbar with `attachment/inline image/` tool